### ISSUE TITLE
Notify miner when new blocks generated

### ISSUE THREAD

**Issue by neal-zhu** [ergoplatform/ergo#1415]
Hi, is there any way to make node notify miner ASAP there is a new block generated instead of polling node in an interval which may be less efficent

**Comment by pragmaxim** [comment]
No, from my experience, such persistent http connections that can be live for weeks or months is not a good idea. And I believe miner should always be the "client" here. It would have to be very sophisticated for it to work ... so currently miners are polling at 100ms intervals and compare the block timestamp.

**Comment by oliverw** [comment]
> No, currently miners are polling at 100ms intervals and compare the block timestamp.

You don't have to tell me the current status quo. I know that myself, thank you. Just because it is that way currently does not mean it should stay that way.

**Comment by kushti** [comment]
There's no notifications API yet
